Prose/ProseLib/Sources/UnreadFeature/SpotlightGroupBoxStyle.swift

```diff
@@ -8,6 +8,10 @@
 import SharedModels
 import SwiftUI
 
+extension GroupBoxStyle where Self == SpotlightGroupBoxStyle {
+    static var spotlight: Self { SpotlightGroupBoxStyle() }
+}
```

Comment:
See the main discussion.

As I see "Refresh" on the top of my page, I guess you already fixed this.

Reply:
I put the comment here, but it's for the file in fact.